public/javascripts/tagLimit.js

```diff
@@ -0,0 +1,12 @@
+// This ensures that only a certain number of tags (3) can be attached to a question
+$(document).ready(function () {
+  $("input[name='tag']").change(function () {
+    var maxAllowed = 3;
```

Comment:
This should be a `const`, since the value is being set only once and referenced elsewhere.